dante/utils.py

```diff
@@ -95,6 +105,9 @@ def getconfig(fn="config.ini", force_reload=False):
     if "DANTE_CONFIG_PATH" in os.environ:
         conf_search_path.append(os.environ["DANTE_CONFIG_PATH"])
     found_files = _config.read(conf_search_path)
```

Comment:
@glciampaglia is 100-107 needed? found_files is not used subsequently

Reply:
Yes these lines are needed, they check if there is a config specified in the environment, which takes precedence. The variable `found_files` is not necessary per se that said if you do want to raise a `RuntimeError` for the (unlikely) situation in which there is no configuration file to read, you could use that. For example something like:
```python
found_files = _config.read(conf_search_path)
if len(found_files) == 0:
     raise RuntimeError("No configuration files found!")
```
Otherwise you can remove found_files however you still need to call `_config.read(...)`